### ISSUE TITLE
Should "quit" be a confirmable command? And how to handle? 

### ISSUE THREAD

**Issue by rocky** [realgud/realgud#125]
Gdb and my debuggers ask for confirmation on quit. They both offer "set confirmation off" so you aren't prompted. Other debuggers quit without confirmation.

I think shortkey's "quit" should also ask for confirmation unless its "set confirmation off" is in effect. But how should this be handled on the backend side? 

Many of my debuggers offer `quit!` which just doesn't prompt. For gdb, I suppose one could do this as the two commands "set confirm off" followed by "quit". Or again add this as macro:

```
define macro  realgud_quit
set confirm off
quit
end
```
 which can go in the growing list profile to run on entering gdb.

@cpitclaudel comments?

**Comment by cpitclaudel** [comment]
I agree. Prompting on the Emacs side when `realgud-safe-mode` is set and then sending an equivalent of `quit!` sounds good.
